shared/modules/consensus_module.go

```diff
@@ -85,4 +90,7 @@ type ConsensusDebugModule interface {
 	SetStep(uint8)
 	SetBlock(*types.Block)
 	SetUtilityContext(UtilityContext)
+
+	SetAggregatedStateSyncMetadata(minHeight, maxHeight uint64, peerAddress string)
```

Comment:
Should this be `Add` because it's only for a single peer?

Reply:
This is used to update peer's aggregated metadata in testing scenarios, to simulate periodic metadata synchronization. So I think in this sense `Set` makes more sense, wdyt?